ImputationPipeline/ProGRESSMultivariateRiskModel.wdl

```diff
@@ -0,0 +1,129 @@
+version 1.0
+
+import "Structs.wdl"
+import "PCATasks.wdl" as PCATasks
+import "ScoreBGE/ScoreBGE.wdl" as ScoreBGE
+
+workflow ProGRESSMultivariateRiskModel {
+    input {
+        File imputed_wgs_vcf
+        File imputed_wgs_vcf_index
+        File exome_gvcf
+        File exome_gvcf_index
+        File prs_weights
+
+        File fam_history
+        String basename
+        
+        File pc_loadings
+        File pc_meansd
+        File pc_sites
+
+        Float prs_beta
+        Float fam_hist_beta
+        Float pc1_beta
+        Float pc2_beta
+
+        Boolean use_ref_alt_for_ids = true
+        String chromosome_encoding = "chrMT"
```

Comment:
Yes I could, although I was thinking given that this is a pipeline with a singular purpose, we could require a defined format rather than running that task (or call caching it) every time. Let me know what you think